opsdroid/tests/test_logging.py

```diff
@@ -21,43 +22,52 @@ def test_set_logging_level():
 
 
 def test_configure_no_logging():
-    config = {"logging": {"path": False, "console": False}}
+    config = {"path": False, "console": False, "rich": False}
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
+
     assert len(rootlogger.handlers) == 1
     assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
     assert rootlogger.handlers[0].level == logging.CRITICAL
 
 
 def test_configure_file_logging():
     with tempfile.TemporaryDirectory() as tmp_dir:
-        config = {
-            "logging": {"path": os.path.join(tmp_dir, "output.log"), "console": False}
-        }
+        config = {"path": os.path.join(tmp_dir, "output.log"), "console": False}
+
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
     assert len(rootlogger.handlers), 2
-    assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
-    assert rootlogger.handlers[0].level, logging.CRITICAL
-    assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
-    assert rootlogger.handlers[1].level == logging.INFO
+    assert isinstance(rootlogger.handlers[0], logging.handlers.RotatingFileHandler)
+    assert isinstance(rootlogger.handlers[1], RichHandler)
+    assert rootlogger.handlers[0].level == logging.INFO
 
 
-def test_configure_file_blacklist(capsys):
+def test_configure_file_logging_only():
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        config = {"path": os.path.join(tmp_dir, "output.log"), "rich": False}
+
+    opsdroid.configure_logging(config)
+    rootlogger = logging.getLogger()
+    assert len(rootlogger.handlers), 1
+    assert isinstance(rootlogger.handlers[0], logging.handlers.RotatingFileHandler)
+    assert rootlogger.handlers[0].level == logging.INFO
+
+
+def test_log_to_file_only(capsys):
     with tempfile.TemporaryDirectory() as tmp_dir:
         config = {
-            "logging": {
-                "path": os.path.join(tmp_dir, "output.log"),
-                "console": False,
-                "filter": {"blacklist": "opsdroid.logging"},
-            }
+            "path": os.path.join(tmp_dir, "output.log"),
+            "console": False,
+            "rich": False,
+            "filter": {"blacklist": "opsdroid.logging"},
         }
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
     assert len(rootlogger.handlers) == 2
     assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
```

Comment:
```suggestion
    assert isinstance(rootlogger.handlers[1], RichHandler)
```

Reply:
That suggestion doesn't make sense, I am setting both console and rich to false and I am testing if we are logging to the file. 

Also after spending time looking into this, the FileHandler creates a streamHandler which uses the file as a stream output so this test seems good